recipe/meta.yaml

```diff
@@ -47,7 +49,8 @@ test:
     - test -f $PREFIX/compute-sanitizer/compute-sanitizer                           # [linux]
     - if not exist %LIBRARY_PREFIX%\compute-sanitizer\compute-sanitizer.exe exit 1  # [win]
     - if not exist %PREFIX%\Scripts\compute-sanitizer.bat exit 1                    # [win]
-    - compute-sanitizer --version
+    - {{ exists }} compute-sanitizer
+    - compute-sanitizer --version                                                   # [build_platform == target_platform]
```

Comment:
As we cannot run `compute-sanitizer` when cross-compiling, have modified the test above to simply check the existence of the command on all platforms. So there is some test that the executable exists even when cross-compiling

Then limited running `compute-sanitizer` to native builds